* grobian@gentoo.org: because we use sandbox this check fails (cannot
  write outside the DESTDIR.  Allowing to write in the
  GNUSTEP_INSTALLATION_DIR means our sandbox is useless, since then
  everything can be written, so prefer to just check the DESTDIR
  location instead.

--- Master/rules.make
+++ Master/rules.make
@@ -107,9 +107,9 @@
 
 ifneq ($(GNUSTEP_INSTALLATION_DIR),)
 internal-check-install-permissions:
-	@if [ -d "$(GNUSTEP_INSTALLATION_DIR)" \
-	      -a ! -w "$(GNUSTEP_INSTALLATION_DIR)" ]; then \
-	  echo "*ERROR*: the software is configured to install itself into $(GNUSTEP_INSTALLATION_DIR)"; \
+	@if [ -d "$(DESTDIR)$(GNUSTEP_INSTALLATION_DIR)" \
+	      -a ! -w "$(DESTDIR)$(GNUSTEP_INSTALLATION_DIR)" ]; then \
+	  echo "*ERROR*: the software is configured to install itself into $(DESTDIR)$(GNUSTEP_INSTALLATION_DIR)"; \
 	  echo "but you do not have permissions to write in that directory:";\
 	  echo "Aborting installation."; \
 	  echo ""; \
